examples/readme.md

```diff
@@ -20,11 +20,12 @@ Our **authenticated** examples are great references if you need help getting sta
 - [Google](https://github.com/mongodb-js/charts-embed-sdk/blob/master/examples/authenticated-google)
 
 If you'd like to build **interactive** charts, we have two examples showing this capability:
-  - [Getting started with click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-basic)
-  - [Interactive filtering using click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-filtering)
+
+- [Getting started with click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-basic)
+- [Interactive filtering using click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-filtering)
```

Comment:
I don't think this is `BETA` anymore, right? @kirangp ?

Reply:
Yes, good pick, will remove the beta label. Thankfully the npm packge in the Interactive Filtering examples is the latest one (not the beta) so it's just the label. :)